tvb_framework/tvb/adapters/creators/siibra_base.py

```diff
@@ -174,7 +174,12 @@ def compute_centroids(region, space):
         centroid = siibra.Point(np.dot(pimg.affine, np.r_[nonzero.mean(0), 1])[:3], space=space)
         centroids.append(centroid)
 
-    return centroids
+    # currently return only the first centroid
+    centroid = centroids[0]
```

Comment:
Does it make sense here to do the for loop above, append multiple `centriod` elements, then only take the first ?
Wodn't be better to just compute the first one, for `label = 1` ?

Reply:
I thought about this, but then decided to leave the for loop to keep the code as similar as possible to the original siibra code. And maybe in the future we decide to retrieve the centroid in a smarter way, not just taking the first one?
But with our current strategy you're right, the for loop is not necessary. Will remove it.

Reply:
Thank you for the explanation. 
Your argument with siibra backwards compatibility makes perfect sense.
In the future I suggest you leave a comment in the code (next to such "controversial" decisions) :)

Reply:
Will do so, thanks for the advice 😊